### DISCUSSION TITLE
Des succès cachés (comme sur Minecraft)

### DISCUSSION THREAD

**Discussion by Gabrielre14** [Crownicles/Crownicles#2495] [Nouvelles idées]
Je viens d'avoir une idée plutôt intéressante en discutant avec Orion des objets sans effet. 

Il pourrait être ajouté au bot des sortes de petits succès qui seront débloqués après avoir fait une action inédite / eu un message inédit, tel que le "Il ne se passe rien" lorsqu'on /bonusjournalier avec un objet sans effet. Ces succès pourraient prendre la forme d'un emoji et d'un petit nom qui seront affichés sous le profil dans une rubrique similaire à celle des joueurs avec 10 badges ou +, où ils s'afficheront tous (emote + nom) si on clique dessous. Cela fera un défi très intéressant pour les joueurs haut niveau qui n'ont plus grand chose à faire XD, surtout qu'ils seront un minimum cachés, c'est-à-dire qu'un joueur lambda qui ne lit pas le code ne sait pas qu'ils existent jusqu'à ce qu'il en découvre un (comme les succès cachés de Minecraft). 

Quelques idées de succès : 
- ❌Un tour pour rien (faire /bonusjournalier avec un objet sans effet)
- 😅C'est pas passé loin (gagner un fight avec 1 PV)
- 😏😏(gagner un fight avec 69 PV)
- 🤕Le plus grand des noobs (perdre face au premier boss)
- 🏥Comment en est-on arrivé là ? (enchaîner toutes les altés du jeu) => ceux qui ont la ref sont forts
- 🤦‍♂️Dommage... (mourir par la branche à -1 PV)
- 🤝Symbiose parfaite (finir un combat à égalité même nombre de PVs, en commençant avec une différence)
- 🐀Quel rat ! (atteindre 500k ou 1M de thune)
- 🏆Outrecuidance suprême (Atteindre le niveau 150) => ceux qui sont pas allé chercher sur internet pour savoir ce que sont sont encore plus forts
- 💪Plus ultra (battre le titan au niveau 120+)
- 🤑Je suis à l'aise financièrement (vendre un item mythique) => ceux qui ont la ref sont des goats
- 🥛 Miam ! (Recevoir un verre de lait d'une déesse ( https://discord.com/channels/429765017332613120/671298370513666058/881120874713976873  => j'ai scrollé pour vous XD ))
- 🐢Cours Forrest, cours ! (perdre une tortue qui se barre en courant ( https://discord.com/channels/429765017332613120/671298370513666058/871024274134147082 => j'ai encore scrollé pour vous))
- 🪓Toujours dans l'abus (infliger plus de 1k de dégats en une attaque)

Si jamais vous avez de meilleurs noms, de meilleures emotes ou tout simplement d'autres idées n'hésitez pas ! Ce doit être un truc très rare, soit extrêmement dur à obtenir, soit que les joueurs ne font jamais.

**Comment by BananePlantain** [comment]
je possède les réfs euh par contre mourir bof

**Comment by NatCrafty** [comment]
Moi je valide, ça serait chouette

**Comment by StrangerIM** [comment]
Je vois ça que maintenant mais très bonne idée ! 

Idées supplémentaires random ( je suis pas sûr de certains ) :
 
- 🍀 **Radin mais chanceux** ( trouver un item légendaire ou plus sans payer ) 

- 👨‍❤️‍👨 **BFF** ( rencontrer le même joueur 2 fois _d'affilée_ )

- 🧐 **My goals are beyond your understanding** ( remplacer un item épique minimum par un item commun dans son inventaire ) 

- 📦 **Marchand de vide-grenier** ( posséder un item de chaque rareté dans sa réserve et son inventaire réunis ) 

- 🐶 **Ami des animaux** ( trouver ~30 pets en une durée déterminée ( le nombre peut changer ))

- 💢 **Enragé** ( collecter au moins 15-20 rage en 1 passage sur l'île )

- ♟️ **Stratège** ( gagner contre un boss de l'île mystérieuse sans rage ni attaque de guilde ) 

- 🎖️**No life** ( posséder tous les badges possibles ) 

- 🐈‍⬛ **Attention à l'échelle** ( loot un chat noir / rencontrer un joueur qui possède un chat noir en pet )

- 🦦 **Loutre** ( rencontrer Cyrpa lol ) 

- 🛸 **Nous venons en paix** ( dresser un alien ) 

- 🎣 **Pêcheur** ( posséder une canne à pêche dans son inventaire ou sa réserve et avoir un poisson en pet ou dans l'abri de sa guilde, en même temps )

**Reply by OrionYp** [reply]
J'aime bien pour la plupart, juste un problème le "no life" qu'on ne peut pas avoir à cause du badge "propriétaire du bot" qu'à drapht.

**Reply by StrangerIM** [reply]
Ah oui j'avais pas connaissance de celui là, mais pas grave y'a qu'à le transformer en ( posséder tous les badges possible à obtenir ) ou un truc du genre

**Reply by romain22222** [reply]
> Ah oui j'avais pas connaissance de celui là, mais pas grave y'a qu'à le transformer en ( posséder tous les badges possible à obtenir ) ou un truc du genre

j'annonce même ainsi il est pas tenable, je pense qu'un succès là dessus n'est pas une bonne idée

**Reply by StrangerIM** [reply]
Ok pas grave

**Reply by BananePlantain** [reply]
le nolife impossible les autres sont validés

**Reply by StrangerIM** [reply]
Ok ok x)

**Comment by NatCrafty** [comment]
Alors je trouve que c’est une bonne idée, par contre pour les 30 pets, a trouver soi même ou on peut aussi acheter ? Et le succès no Life je propose que l’on le gagne quand on obtient tout les succès cachés.

**Reply by StrangerIM** [reply]
Je pensais à trouver soi même 
Et on abandonne le no Life

**Reply by NatCrafty** [reply]
Personnellement je pense que ça serait mieux de prendre en compte aussi les pets achetés pour ce succès, ou alors d’augmenter le nombre à trouver et de faire sans durée fixe

**Reply by StrangerIM** [reply]
Si on prend en compte les pets achetés aussi, le succès est à la portée de tout le monde
Et si on défini pas de durée maximum pour l'accomplir tout le monde finira par l'avoir

**Reply by NatCrafty** [reply]
Alors juste on met un temps un minimum possible

**Comment by BananePlantain** [comment]
on pourrait mettre des succès avec des tiers du style chépa gagner 10/25/100 combats (mais pas avec des combats sinon c'est facile)

**Reply by StrangerIM** [reply]
Bonne idée, genre "🐶 Ami des animaux I, II, III" ou "💢 Enragé I, II, III"

**Comment by QQtin3** [comment]
J'adore cette idée vous êtes également très créatifs, le problème majeur que je vois à la suggestion est : les badges, à mon sens c'est un système très proche qui, oui n'est pas "prestigieux" dans le sens où ce sont des sortes de distinctions pour certaines actions (ou pas d'ailleurs), j'imagine qu'on pourrait laisser les deux systèmes mais c'est quelque chose sur lequel il pourrait être intéressant de réfléchir

**Reply by StrangerIM** [reply]
De ce que je sais il n'est question que d'un seul badge, qui lorsqu'on clique dessus affiche les succès complétés par le joueur

**Reply by QQtin3** [reply]
Oui donc c'est pas vraiment un badge, juste une réaction sous le profil qui afficherait ça ? Un peu comme lorsque l'on a + de (15 badges ? Je sais plus de combien est le threshhold)

**Reply by StrangerIM** [reply]
Yep c'est ça
